wolfssl/wolfcrypt/wc_port.h

```diff
@@ -38,14 +38,14 @@
         #ifndef WIN32_LEAN_AND_MEAN
             #define WIN32_LEAN_AND_MEAN
         #endif
-    #ifndef WOLFCRYPT_ONLY
+    #ifndef WOLFSSL_SGX
```

Comment:
This was added in the initial SGX pull request to avoid including Windows system header files. In the initial SGX pull request it should have been WOLFSSL_SGX to keep the status quo for previous WOLFCRYPT_ONLY builds on Windows. The SGX build does not need the Windows system header files because it includes SGX header files instead.